Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8-buster
+WORKDIR /opt/proxstar
+RUN apt-get update -y && apt-get install -y python3-dev libldap2-dev libsasl2-dev ldap-utils tox lcov valgrind
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+COPY start_worker.sh start_scheduler.sh .
+COPY .git ./.git # TODO (willnilges): The app crashes without it. Remove if you can.
```

Comment:
This is needed for the version tag at the bottom of the page

Reply:
Is there a specific file i can `COPY` without having to do the entire directory?